build/package.json

```diff
@@ -71,23 +71,23 @@
     "rollup-plugin-commonjs": "^10.1.0",
     "rollup-plugin-node-resolve": "^5.2.0",
     "source-map": "0.6.1",
+    "through2": "^4.0.2",
     "tmp": "^0.2.1",
-    "typescript": "^4.8.0-dev.20220518",
-    "vsce": "2.8.0",
     "vscode-universal-bundler": "^0.0.2"
   },
   "scripts": {
-    "compile": "tsc -p tsconfig.build.json",
-    "watch": "tsc -p tsconfig.build.json --watch",
-    "npmCheckJs": "tsc --noEmit"
+    "compile": "../node_modules/.bin/tsc -p tsconfig.build.json",
+    "watch": "../node_modules/.bin/tsc -p tsconfig.build.json --watch",
+    "npmCheckJs": "../node_modules/.bin/tsc --noEmit"
   },
   "optionalDependencies": {
     "tree-sitter": "https://github.com/joaomoreno/node-tree-sitter/releases/download/v0.20.0/tree-sitter-0.20.0.tgz",
     "tree-sitter-typescript": "^0.20.1",
     "vscode-gulp-watch": "^5.0.3"
   },
-  "resolutions": {
-    "json-schema": "0.4.0",
-    "jsonwebtoken": "9.0.0"
+  "dependencies": {
+    "@actions/core": "1.2.6",
```

Comment:
We shouldn't have dependencies in this package.json (it's only used during build time), put them in devDependencies instead.